airflow_munchkin/block_generator/hook_generator.py

```diff
@@ -103,7 +102,7 @@ def convert_path_parameter_block_to_individual_parameters(
     required_parameters: Dict[str, ParameterBlock] = {}
     call_args = []
 
-    for name in path_info.args:
+    for name in path_info.args:  # type: ignore
```

Comment:
We don't use a `type: ignore` in this project. Can you fix type hint instead?

Reply:
Sure, I will fix it.

Reply:
Moreover, during deletion of mine "ignores" I also deleted some additional `type: ignore` which seems to be unnecessary  🕺